packages/gestalt/src/Video.js

```diff
@@ -376,6 +382,10 @@ export default class Video extends PureComponent<Props, State> {
     // Set the initial playback rate
     this.setPlaybackRate(playbackRate);
 
+    if (startTime) {
```

Comment:
We probably also want to add logic in componentDidUpdate to listen for any changes to `startTime`, yes? Or would we want to ignore those changes? I could see a use case where the start time is changed outside of Video, and we'd want to jump to that new start time — but let me know if that's not the case.

Reply:
Yeah, I thought about this for a bit. It's probably the best way to avoid outside components updating the `currentTime` if they need to do it once video play is underway. Otherwise we'd need to expose `seek` API for the wrapper

Reply:
If we want a constant listener, maybe we should call this `currentTime` instead of `startTime` ?

Reply:
> If we want a constant listener, maybe we should call this `currentTime` instead of `startTime` ?

I was initially thinking that keeping closer to the native API would be better, but since `currentTime` is read-write, we'd have to keep it in sync, which sounds like a bit of a headache. (And using the native attribute name without replicating the native behavior would be confusing.)

> It's probably the best way to avoid outside components updating the `currentTime` if they need to do it once video play is underway. Otherwise we'd need to expose `seek` API for the wrapper

Ok, if that's something we want to avoid, that's fine. From the [fastSeek docs](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/fastSeek) it sounds like this would be the prop to use to programmatically to a future time, but I'm also fine with leaving out that functionality for now if it's not needed:
> Note: If you need to seek with precision, you should set [HTMLMediaElement.currentTime](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/currentTime) instead.

Reply:
Yeah fastSeek is cool but no chrome support 😢 . I'll update the lifecycle methods for startTime and hopefully that's clear enough for it to stick. Having a hard time envisioning a scenario where someone needs to programatically set `currentTime` multiple times without user interaction so hopefully viewing it as a startTime only is good enough